### DISCUSSION TITLE
新起点 | Super Blog

### DISCUSSION THREAD

**Discussion by giscus** [SuperPung/Blog#7] [General]
# 新起点 | Super Blog

新的样子，新的开始。

https://superpung.com/new-ui/

**Comment by superpung** [comment]
Hello, world!

**Comment by abeelan** [comment]
你好，请问博客的搭建方式和主题方便分享下吗？很赞呀

**Reply by superpung** [reply]
Hexo+NexT 魔改，但目前还有一些没解决的问题，之后会考虑开源 😄

**Comment by wjwei-handsome** [comment]
你好，可以修改comment.css 在其中添加media的dark模式，可以参考[https://github.com/giscus/giscus/blob/main/styles/themes/preferred_color_scheme.css](https://github.com/giscus/giscus/blob/main/styles/themes/preferred_color_scheme.css)，可以上传到自己的cdn，我是这样实现的：https://github.com/wjwei-handsome/wjwei-docusaurus/blob/main/src/components/GiscusComponent.js 
有更好的方式欢迎交流~

**Reply by superpung** [reply]
感谢！我之后看一下 🥳
